packages/waku/src/lib/utils/buffer.ts

```diff
@@ -0,0 +1,16 @@
+export const parseFormData = async (
+  buffer: ArrayBuffer,
+  contentType: string,
+): Promise<FormData> => {
+  const response = new Response(buffer, {
+    headers: {
+      'Content-Type': contentType,
```

Comment:
Maybe it works with this:
```suggestion
      'content-type': contentType,
```